cli/docker-machine.go

```diff
@@ -0,0 +1,26 @@
+package run
+
+import (
+	"os"
+	"os/exec"
+	"strings"
+)
+
+func getDockerMachineIp() (string, error) {
+	machine := os.ExpandEnv("$DOCKER_MACHINE_NAME")
+
+	if machine == "" {
+		return "", nil
+	}
+
+	dockerMachinePath, err := exec.LookPath("docker-machine")
+	if err != nil {
+		return "", err
+	}
+
+	out, err := exec.Command(dockerMachinePath, "ip").Output()
```

Comment:
Use the `machine` variable from L10

Reply:
I don't think 'machine' here is necessary. Without it, the machine name will be picked up from the environment variable, which will end up being the same.